app/assets/javascripts/models/attachment_options.js

```diff
@@ -0,0 +1,63 @@
+export default class AttachmentOptions {
+
+  constructor({ refreshCallback }) {
+    this.refreshTime = 3600000;
+    this.timestampVerifierId;
+    this.refreshCallback = refreshCallback;
+
+    this.getOptions();
```

Comment:
Prefer to do that outside the constructor, there should be no side-effect inside constructors. What do you think about having a `fetch()` method that you call after you instantiate it? You count instantiate it in the components constructor (like you're doing already) and then call the `fetch()` inside the `componentWillMount`.